install/install.php

```diff
@@ -288,102 +285,70 @@ public function __construct($dbh)
-                Toolbox::createSchema($_SESSION["glpilanguage"]);
-                echo "<p>" . __('OK - database was initialized') . "</p>";
-                $next_form();
-            } else { // can't create config_db file
-                echo "<p>" . __('Impossible to write the database setup file') . "</p>";
-                $prev_form($host, $user, $password);
-            }
-        } else { // try to create the DB
-            if ($link->query("CREATE DATABASE IF NOT EXISTS `" . $newdatabasename . "`")) {
-                echo "<p>" . __('Database created') . "</p>";
-
-                $select_db = $link->select_db($newdatabasename);
-                $success = false;
-                if ($select_db) {
-                    $success = DBConnection::createMainConfig(
-                        $host,
-                        $user,
-                        $password,
-                        $newdatabasename,
-                        use_timezones: $timezones_requirement->isValidated(),
-                        log_deprecation_warnings: false,
-                        use_utf8mb4: true,
-                        allow_datetime: false,
-                        allow_signed_keys: false
-                    );
-                }
+    } else {
+        $databasename = $link->real_escape_string($databasename);
+    }
 
-                if ($success) {
-                    echo "<p>" . __('Initializing database tables and default data...') . "</p>";
-                    Toolbox::createSchema($_SESSION["glpilanguage"]);
-                    echo "<p>" . __('OK - database was initialized') . "</p>";
-                    $next_form();
-                } else { // can't create config_db file
-                    echo "<p>" . __('Impossible to write the database setup file') . "</p>";
-                    $prev_form($host, $user, $password);
-                }
-            } else { // can't create database
-                echo __('Error in creating database!');
-                echo "<br>" . sprintf(__('The server answered: %s'), $link->error);
-                $prev_form($host, $user, $password);
-            }
-        }
-    } else { // no db selected
-        echo "<p>" . __("You didn't select a database!") . "</p>";
+    if (!$link->select_db($databasename)) {
+        echo __('Impossible to use the database:');
+        echo "<br>" . sprintf(__('The server answered: %s'), $link->error);
         $prev_form($host, $user, $password);
+        return;
     }
 
-    $link->close();
+    $success = DBConnection::createMainConfig(
+        $host,
+        $user,
+        $password,
+        $databasename,
+        use_timezones: $timezones_requirement->isValidated(),
+        log_deprecation_warnings: false,
+        use_utf8mb4: true,
+        allow_datetime: false,
+        allow_signed_keys: false
+    );
+
+    if ($success) {
+        echo "<p>" . __('Initializing database tables and default data...') . "</p>";
```

Comment:
We probably need string escaping here (for all `echo`)
```suggestion
        echo "<p>" . __s('Initializing database tables and default data...') . "</p>";
```

Reply:
I planned to do it in another PR, with all other install/update scripts echo.

Reply:
ok